static/app/views/performance/newTraceDetails/index.tsx

```diff
@@ -251,13 +313,21 @@ type TraceViewContentProps = {
   organization: Organization;
   traceEventView: EventView;
   traceSlug: string;
-  traces: TraceSplitResults<TraceFullDetailed> | null;
+  traceSplitResult: TraceSplitResults<TraceFullDetailed> | null;
 };
 
 function TraceViewContent(props: TraceViewContentProps) {
-  const {organization, traces, traceSlug, location, traceEventView, metaResults} = props;
+  const {
```

Comment:
Just access props, these short hands are not valuable as we lose the notion of data flow in the component body as everything looks locally defined. Accessing via props.prop is a bit more verbose, but easier to see where the data originates from